src/main/java/roomescape/service/AuthService.java

```diff
@@ -0,0 +1,57 @@
+package roomescape.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import roomescape.domain.User;
+import roomescape.dto.auth.AuthCheckResponse;
+import roomescape.dto.auth.AuthLoginRequest;
+import roomescape.exception.ErrorCode;
+import roomescape.exception.custom.AuthorizationException;
+import roomescape.exception.custom.UserNotFoundException;
+import roomescape.jwt.JwtTokenProvider;
+import roomescape.repository.AuthRepository;
+
+@Service
+@Transactional
+public class AuthService {
+
+    private final AuthRepository authRepository;
+
+    private final JwtTokenProvider jwtTokenProvider;
+
+    private static final Logger log = LoggerFactory.getLogger(AuthService.class);
+
+    @Value("${security.jwt.token.secret-key}")
+    private String secretKey;
+
+    public AuthService(AuthRepository authRepository, JwtTokenProvider jwtTokenProvider) {
+        this.authRepository = authRepository;
+        this.jwtTokenProvider = jwtTokenProvider;
+    }
+
+    public String authLogin(AuthLoginRequest request) {
+        User user = authRepository.findUserByEmail(request.getEmail())
+                .orElseThrow(() -> {
+                    throw new UserNotFoundException(ErrorCode.USER_NOT_FOUND,"존재하지 않는 회원입니다. 다시 입력해주세요.");
+                });
+        checkInvalidLogin(user, request);
+        return jwtTokenProvider.createToken(request.getEmail());
+
+    }
+
+    private void checkInvalidLogin(User user, AuthLoginRequest request) {
+        if (!user.getPassword().equals(request.getPassword())) {
```

Comment:
```suggestion
        if (!user.checkPassword()) {
```
와 같이 유저 객체에게 비밀번호가 같은지 물어볼 수 있지 않을까요~?